clojure.c

```diff
@@ -0,0 +1,163 @@
+/**
+ *   Copyright (c) 2015, Miloslav Nenadál <nenadalm@gmail.com>
+ *
+ *   This source code is released for free distribution under the terms of the
+ *   GNU General Public License.
+ *
+ *   This module contains code for generating tags for the PHP scripting
+ *   language.
+ */
+
+#include <string.h>
+
+#include "general.h"
+
+#include "parse.h"
+#include "read.h"
+#include "vstring.h"
+#include "entry.h"
+
+typedef enum {
+	K_FUNCTION,
+	K_NAMESPACE
+} clojureKind;
+
+static kindOption ClojureKinds[] = {
+	{TRUE, 'f', "function", "functions"},
+	{TRUE, 'n', "namespace", "namespaces"}
+};
+
+static vString *CurrentNamespace;
+
+static int isNamespace (const unsigned char *strp)
+{
+	return strp[1] == 'n' && strp[2] == 's';
+}
+
+static int isFunction (const unsigned char *strp)
+{
+	return strp[1] == 'd' && strp[2] == 'e' && strp[3] == 'f' && strp[4] == 'n';
```

Comment:
shouldn't you make sure the character next to "defn" is either a space or the end of the line?  kinda like you do in `isQuote()` below.
Same would apply to `isNamespace()` above.